fullnode/src/main.rs

```diff
@@ -130,6 +130,12 @@ fn main() {
 
     let matches = App::new("fullnode")
         .version(crate_version!())
+        .arg(
+            Arg::with_name("entry_stream")
+                .long("entry-stream")
+                .takes_value(true)
```

Comment:
how about adding a `.value_name("UNIX DOMAIN SOCKET")` or something, for more better `-h` text